spec/models/season_spec.rb

```diff
@@ -190,4 +190,30 @@
     end
   end
 
-end
+  ######## Testing Season Switch #############
+  describe 'switch phase' do
```

Comment:
Can you turn that outer `describe` into a `context` and then wrap the inner `it` blocks into a `describe #method_name` ?

Example:
```ruby
context 'switching phases' do
  describe '#fake_application_phase' do
    it 'timeshifts the application phase to today' do
      # ...
    end
  end

  describe '#fake_coding_phase' do
    # ...
  end
end
```